dotcom-rendering/scripts/webpack/browser.js

```diff
@@ -94,6 +94,7 @@ module.exports = ({ isLegacyJS }) => ({
 											},
 									  ],
 							],
+							compact: true,
```

Comment:
This silences some warnings we saw in the console. See: https://stackoverflow.com/questions/29576341/what-does-the-code-generator-has-deoptimised-the-styling-of-some-file-as-it-e